shared/tests/serializer/content-loader-serializer.js

```diff
@@ -1,19 +1,17 @@
-module.exports = {
-  test(value) {
-    return (value && typeof value === 'string' && /url\(/.test(value)) || /-aria$/.test(value) || /-diff$/.test(value);
-  },
+export function test(value) {
+  return (value && typeof value === 'string' && /url\(/.test(value)) || /-aria$/.test(value) || /-diff$/.test(value);
+}
 
-  print(value, serialize) {
-    if (/-aria$/.test(value)) {
-      value = 'TITLE_ARIA_ID';
-    }
-
-    value = value.replace(/url\((.*)\)/, 'STYLE-URL');
+export function print(value, serialize) {
+  if (/-aria$/.test(value)) {
+    value = 'TITLE_ARIA_ID';
+  }
 
-    if (/-diff$/.test(value)) {
-      value = 'DIFF_ID';
-    }
+  value = value.replace(/url\((.*)\)/, 'STYLE-URL');
 
-    return serialize(value);
+  if (/-diff$/.test(value)) {
+    value = 'DIFF_ID';
   }
-};
+
+  return serialize(value);
+}
```

Comment:
Could you please reset those changes and rename the file to use the `.cjs` extension?